test/functional/ui/screen.lua

```diff
@@ -492,7 +492,7 @@ function Screen:_wait(check, flags)
       end
     elseif success_seen and #args > 0 then
       failure_after_success = true
-      --print(require('inspect')(args))
+      --print(require('vim.inspect')(args))
```

Comment:
we can just use `inspect` here, as it is imported as `local inspect` anyway (+ two more cases).

Reply:
Good catch, fixed.

(not really a fan of having commented code like that in the first place, but might be helpful at least in some case(s))

Reply:
Maybe it can be just removed, some of it could be older than `screen:redraw_debug()` and the like.

Reply:
Yeah.. but let's keep it for now.